<launch>

    <node pkg="MODT" type="ST_GenRawTrainingData" name="ST_GenRawTrainingData" ns="iMiev" output="screen">
		<param name="program_mode_" value="0"/>
		<param name="DATMO_model_path" value="$(find MODT)/data_and_model/MODT.model"/>
		<param name="DATMO_scale_path" value="$(find MODT)/data_and_model/MODT.range"/>
		<param name="laser_frame_id" value="iMiev/front_bottom_lidar"/>
		<param name="odom_frame_id" value="iMiev/odom"/>
		
		<!--when doing the derived data generation, the size value should be bigger than this value-->
		<param name="speed_threshold" value= "3.0"/>
		<param name="smallest_size_threshold" value= "2.0"/>
    </node>
</launch>

